unit_test/cmockalib/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
     ADD_COMPILE_OPTIONS(${CMOCKA_FLAGS} -DHAVE_SIGNAL_H)
 elseif(CMAKE_SYSTEM_NAME MATCHES "Windows")
     ADD_COMPILE_OPTIONS(${CMOCKA_FLAGS} -DHAVE_VSNPRINTF -DHAVE_SNPRINTF)
+    if((TOOLCHAIN STREQUAL "VS2015") OR (TOOLCHAIN STREQUAL "VS2019") OR (TOOLCHAIN STREQUAL "VS2022"))
```

Comment:
Isn't this warning only on VS2022?

Reply:
No. The unicode warning also occurs when I using VS2019 to build the project.

C4819 occurs when you compile an ANSI source file on a system using a codepage that can't represent all characters in the file. more details in [here](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4819?view=msvc-140)